vscode-ext/package.json

```diff
@@ -3,7 +3,7 @@
 	"displayName": "mirrord",
 	"description": "Mirror live traffic from your Kubernetes cluster to your local debugged process.",
 	"publisher": "MetalBear",
-	"version": "2.4.1",
+	"version": "1.0.0",
```

Comment:
Why did you change the version?

Reply:
That happened because I was playing with `vsce package`, and somehow I missed it while looking at changes.